samples/interop-customvision-textmsg-uwpapp/Documentation/Configuring the IoT Edge Device.MD

```diff
@@ -0,0 +1,108 @@
+# Text Messaging & Custom Vision Interop Samples
+## Progress
+
+- [x] [Introduction](../README.md)  
+- [x] [Step 1 - Setup Development Environment](./Setup%20DevVM.MD)   
+- [x] [Step 2 - Setup Azure Resources](./Setup%20Azure%20Resources.MD)  
+- [x] [Step 3 - Setup Azure IoT Edge for Linux on Windows](./Setup%20Azure%20IoT%20Edge%20for%20Linux%20on%20Windows.MD)
+- [x] [Step 4 - Develop and publish the IoT Edge Linux module](./Develop%20and%20publish%20the%20IoT%20edge%20Linux%20module.MD)  
+- [x] [Step 5 - Create Certificates for Authentication](./Create%20Certificates%20for%20Authentication.MD)  
+- [x] [Step 6 - Develop the Windows C# UWP Application](./Develop%20the%20Windows%20C%23%20UWP%20Application.MD)
+- [ ] **Step 7 - Configuring the IoT Edge Device**  
+- [ ] [Step 8 - Run samples](./Run%20samples.MD) 
+- [ ] [Troubleshooting](./Troubleshooting.MD) 
+---
+
+# Step 7: Configuring the IoT Edge Device.
+In this part we configure the Azure IoT Edge for Linux instance on the IoT edge device.
+
+
+1. **Create Certificates folder inside the EFLOW VM**
+    * Use SSH to log in to the EFLOW VM environment.  
+
+        ```powershell
+        Ssh-EflowVm
+        ``` 
+
+    * Inside the EFLOW VM environment, use mkdir to create the Certificates folder  
+
+        ```powershell
+        mkdir ~/certs/
+        ``` 
+
+2. **Copy Certificates to EFLOW VM environment**
+
+   * Use PowerShell to get the EFLOW VM IP address.  
+
```

Comment:
how about moving this step (getting vm's ip address) to the first step. It needs to be run on the host device. We might need to mention it explicitly.

Reply:
I will add the mention to PowerShell on host - regarding moving to the first step, I like steps to be sequential, so if we put that on the first step, they wont use it until 2.1